lms/models/user.py

```diff
@@ -36,8 +36,8 @@ class User(CreatedUpdatedMixin, Base):
     user_id: Mapped[str] = mapped_column(sa.Unicode, nullable=False)
     """The user id provided by the LTI parameters."""
 
-    roles = sa.Column(sa.Unicode, nullable=True)
-    """The roles provided by the LTI parameters."""
+    legacy_roles = sa.Column(sa.Unicode, nullable=True)
```

Comment:
I think for removing columns this is the best "backwards migration", keep the column around and if something breaks downstream (report? dashboards?) we have a way to either rollback or do a quick fix.